autoinstaller/install.sh

```diff
@@ -62,12 +305,22 @@ VAI_install_xbps_keys() {
 }
 
 VAI_install_base_system() {
+    base_pkgs="base-system grub grub-x86_64-efi grub-i386-efi";
```

Comment:
ideally this would be an intelligent decision to avoid installing unnecessary packages.